shared/common-adapters/keyboard-avoiding-view.native.tsx

```diff
@@ -170,16 +168,15 @@ class KeyboardAvoidingView extends React.Component<Props, State> {
         },
       })
     }
-    this._bottom = height
-    this.setState({bottom: height + (this.props.extraPadding ?? 0)})
+    this._setBottom(height)
   }
 
-  componentDidUpdate(_: Props, prevState: State): void {
-    const enabled = this.props.enabled ?? true
-    if (enabled && this._bottom !== prevState.bottom) {
-      this.setState({bottom: this._bottom})
-    }
-  }
+  // componentDidUpdate(_: Props, prevState: State): void {
```

Comment:
we stash the raw value in bottom but set state including an offset so lets maintain that. we never enable/disable this component